pmpsui/pmps.py

```diff
@@ -10,40 +10,38 @@
 from pydm.widgets.channel import PyDMChannel
 from qtpy import QtCore
 
-from beamclass_table import install_bc_setText
-from tooltips import (get_mode_tooltip_lines, get_tooltip_for_bc,
-                      setup_combobox_tooltip)
-from utils import BackCompat, morph_into_vertical
-from widgets import EvByteIndicator
+from pmpsui.beamclass_table import install_bc_setText
+from pmpsui.tooltips import (get_mode_tooltip_lines, get_tooltip_for_bc,
+                             setup_combobox_tooltip)
+from pmpsui.utils import BackCompat, morph_into_vertical
+from pmpsui.widgets import EvByteIndicator
 
 logger = logging.getLogger(__name__)
 
 
-def make_parser():
-    parser = argparse.ArgumentParser(
-        description='Display the PMPS diagnostic tool inside of PyDM.',
-        prog='pydm pmps.py',
-    )
-    parser.add_argument(
-        '--no-web',
-        action='store_true',
-        help='Disable the grafana web view tab.',
-    )
-    return parser
-
-
 class PMPS(Display):
     new_mode_signal = QtCore.Signal(str)
 
     def __init__(self, parent=None, args=None, macros=None):
-        parser = make_parser()
-        self.user_args = parser.parse_args(args=args or [])
+        # self.user_args = args
+        # Assumes args are passed as list of [no_web, log_level] as in main.py
+        self.user_args = argparse.Namespace(no_web=args[0], log_level=args[1])
```

Comment:
Maybe this can be done a bit more cleanly later, I wonder how much we can change the args in a `Display` subclass...